appengine/flexible/tasks/README.md

```diff
@@ -42,17 +42,18 @@ for more information.
 
 ## Creating a queue
 
-To create a queue using the Cloud SDK, use the provided queue.yaml:
+To create a queue using the Cloud SDK, use the following gcloud command:
 
-    gcloud app deploy queue.yaml
+    gcloud alpha tasks queues create-app-engine-queue "my-appengine-queue"
```

Comment:
you don't need to put the queue ID in quotes

Reply:
Done